synapse/rest/client/v1/admin.py

```diff
@@ -63,6 +65,116 @@ def on_GET(self, request, user_id):
         defer.returnValue((200, ret))
 
 
+class UserRegisterServlet(ClientV1RestServlet):
+    PATTERNS = client_path_patterns("/admin/register")
+    SALT_TIMEOUT = 60
```

Comment:
can we have a comment to explain what this does?

Reply:
done